docs/design.md

```diff
@@ -56,8 +56,8 @@ load paths to include the folders with the OpenTelemetry .NET SDK and the instru
 injected into the application.
 
 - [**Managed Profiler**](../src/OpenTelemetry.AutoInstrumentation):
-contains the code setting up the OpenTelemtry .NET SDK and configured instrumentations.
-Plus, support code to run and implement bytecode instrumentations. Optionally, the code
+contains the code setting up the OpenTelemetry .NET SDK and configured instrumentations.
+Additionally, support code to run and implement bytecode instrumentations. Optionally, the code
 setting-up the OpenTelemetry .NET SDK can be left to the application by setting the
 environment variable `OTEL_DOTNET_AUTO_LOAD_AT_STARTUP` to `false`.
```

Comment:
```suggestion
that sets up the OpenTelemetry .NET SDK can be left to the application by setting the
environment variable `OTEL_DOTNET_AUTO_LOAD_AT_STARTUP` to `false`.
```

What does "left to the application" mean in this situation?

Reply:
That the application can bootstrap/configure the tracer.